your-turn/hamdi.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../main.css" rel="stylesheet">
+    <title>Hamdi Here!</title>
+</head>
+<body> 
+    <h1>Hamdi</h1>
+    <img src="https://legionofart.com/uploads/blackgirlmagic_508971.jpeg" class="side profile picture"> 
+
+    <div class="container">
+        <ul>
+            <li>My favorite color is purple</li>
+            <li>My favorite book is A Thousand Splendid Suns</li>
+            <li>My birthday is on a major holiday. Take a guess!</li>
```

Comment:
Christmas? 👀

Reply:
Yup! 

Also, I made changes, saved them, and pushed it to the origin. I included the Paris image. It looks like it's showing up. Should I pull a new request or can you see those changes to be able to merge them? Thank you for your feedback!!

Reply:
Nice!
Your changes look great! Since you already have an open pull request, you do not need to make a new one :)